tests/epics/adaravis/test_aravis.py

```diff
@@ -123,6 +123,7 @@ async def test_can_collect(
         "dataset": "/entry/data/data",
         "swmr": False,
         "multiplier": 1,
+        "chunk_size": 0,
```

Comment:
Does `chunk_size == 0` means that the leftmost dimension is not chunked (i.e. single chunk)?

Reply:
This is how `TiledWriter` and `Consolidators` treat it now.